express-api/src/typeorm/Migrations/1710191053571-AddBuildingRelationsView.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddBuildingRelationsView1710191053571 implements MigrationInterface {
+  name = 'AddBuildingRelationsView1710191053571';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE MATERIALIZED VIEW "building_relations" AS 
+    SELECT b.id AS "building_id", b.pid, b.pin, p.id as "parcel_id"
+    FROM building b
+    LEFT OUTER JOIN parcel p
+    ON b.pid = p.pid OR b.pin = p.pin;
+  `);
+    await queryRunner.query(
```

Comment:
Is writing directly to this table correct? Seems like something that should be auto-generated.

Reply:
This was what the TypeORM package generated. I didn't manually write this.